var/spack/repos/builtin/packages/r-circstat/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RCircstat(RPackage):
+    """Circular Statistics, from "Topics in Circular Statistics" (2001)"""
+
```

Comment:
Please add the description as well. I know in this case it is similar to the title.